File: codigos/1 - estrutura sequencial/1019 - Conversao de Tempo/Main.java
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int segundos = new Scanner(System.in).nextInt();
        int[] tempos = {3600, 60, 1};
        ArrayList<String> saidas = new ArrayList<>();
        for (int tempo:tempos) {
            int qntd_tempo_temp;
            if (segundos >= tempo){
                qntd_tempo_temp = segundos/tempo;
                saidas.add(String.valueOf(qntd_tempo_temp));
                segundos -= qntd_tempo_temp * tempo;
            } else {
                qntd_tempo_temp = 0;
                saidas.add(String.valueOf(qntd_tempo_temp));
            }
        }
        System.out.println(String.join(":", saidas));
    }
}

File: codigos/1 - estrutura sequencial/1019 - Conversao de Tempo/Main.py
from math import floor

segundos = int(input())
saidas: list[int] = []
for tempo in (tempos:=[3600, 60, 1]):
    qntd_tempo_temp = floor(segundos/tempo) if segundos >= tempo else 0
    saidas.append(qntd_tempo_temp)
    segundos -= qntd_tempo_temp * tempo

print(":".join(map(str, saidas)))